<launch>
  <arg name="data_dir" /> 
  <param name="ec_cluster_tolerance" value="0.03" />
  <param name="ec_min_cluster_size" value="700" />
  <param name="ec_max_cluster_size" value="10000" />
  <param name="distance_above_plane" value="0.01" />
  <param name="crop_min_x" value="-1" />
  <param name="crop_min_y" value="-1" />
  <param name="crop_min_z" value="0" />
  <param name="crop_max_x" value="1.05" />
  <param name="crop_max_y" value="1.0" />
  <param name="crop_max_z" value="1.2" />

  <node pkg="tf" type="static_transform_publisher" name="tf_static" args="0 0 0 0 0 0 base_link mock_point_cloud 100" />

  <node pkg="perception" type="pointcloud_demo" name="point_cloud_demo" args="$(arg data_dir)" output="screen">
    <remap from="cloud_in" to="mock_point_cloud" />
  </node>
</launch>  
